legacy-docs/docs/r/l3_ext_subnet.html.markdown

```diff
@@ -48,6 +49,7 @@ Manages ACI External EPG Subnet
 
 * `external_network_instance_profile_dn` - (Required) Distinguished name of the parent External Network Instance Profile object.
 * `ip` - (Required) ip of the External EPG Subnet object.
+* `name` - (Optional) Name of the External EPG Subnet object.
```

Comment:
Can you quickly look at the other sentences consistency and adjust capitalisation. IP and annotation are for instance lower case.